content/en/docs/tasks/traffic-management/egress/egress-gateway-tls-origination/snips.sh

```diff
@@ -300,212 +294,20 @@ EOF
-kubectl exec "$(kubectl get pod -l app=sleep -o jsonpath={.items..metadata.name})" -c sleep -- curl -k --resolve nginx.example.com:443:1.1.1.1 https://nginx.example.com
-}
-
-! read -r -d '' snip_deploy_a_container_to_test_the_nginx_deployment_5_out <<\ENDSNIP
-...
-<html>
-<head><title>400 No required SSL certificate was sent</title></head>
-<body>
-<center><h1>400 Bad Request</h1></center>
-<center>No required SSL certificate was sent</center>
-...
-</body>
-</html>
-ENDSNIP
-
-snip_redeploy_the_egress_gateway_with_the_client_certificates_1() {
-kubectl create -n istio-system secret tls nginx-client-certs --key nginx.example.com/4_client/private/nginx.example.com.key.pem --cert nginx.example.com/4_client/certs/nginx.example.com.cert.pem
-kubectl create -n istio-system secret generic nginx-ca-certs --from-file=nginx.example.com/2_intermediate/certs/ca-chain.cert.pem
+kubectl apply -f samples/sleep/sleep.yaml
 }
 
-snip_redeploy_the_egress_gateway_with_the_client_certificates_2() {
-cat > gateway-patch.json <<EOF
-[{
-  "op": "add",
-  "path": "/spec/template/spec/containers/0/volumeMounts/0",
-  "value": {
-    "mountPath": "/etc/istio/nginx-client-certs",
-    "name": "nginx-client-certs",
-    "readOnly": true
-  }
-},
-{
-  "op": "add",
-  "path": "/spec/template/spec/volumes/0",
-  "value": {
-  "name": "nginx-client-certs",
-    "secret": {
-      "secretName": "nginx-client-certs",
-      "optional": true
-    }
-  }
-},
-{
-  "op": "add",
-  "path": "/spec/template/spec/containers/0/volumeMounts/1",
-  "value": {
-    "mountPath": "/etc/istio/nginx-ca-certs",
-    "name": "nginx-ca-certs",
-    "readOnly": true
-  }
-},
-{
-  "op": "add",
-  "path": "/spec/template/spec/volumes/1",
-  "value": {
-  "name": "nginx-ca-certs",
-    "secret": {
-      "secretName": "nginx-ca-certs",
-      "optional": true
-    }
-  }
-}]
-EOF
+snip_deploy_a_mutual_tls_server_7() {
+kubectl apply -f <(istioctl kube-inject -f samples/sleep/sleep.yaml)
 }
 
-snip_redeploy_the_egress_gateway_with_the_client_certificates_3() {
-kubectl -n istio-system patch --type=json deploy istio-egressgateway -p "$(cat gateway-patch.json)"
+snip_mount_client_certificates_to_the_egress_gateway_1() {
+kubectl create secret tls istio-egressgateway-certs --key client.example.com.key --cert client.example.com.crt -n istio-system
+kubectl create secret generic istio-egressgateway-ca-certs --from-file=example.com.crt -n istio-system
 }
 
-snip_redeploy_the_egress_gateway_with_the_client_certificates_4() {
-kubectl exec -n istio-system "$(kubectl -n istio-system get pods -l istio=egressgateway -o jsonpath='{.items[0].metadata.name}')" -- ls -al /etc/istio/nginx-client-certs /etc/istio/nginx-ca-certs
+snip_mount_client_certificates_to_the_egress_gateway_2() {
+kubectl exec -n istio-system $(kubectl -n istio-system get pods -l istio=egressgateway -o jsonpath='{.items[0].metadata.name}') -- ls -al /etc/istio/egressgateway-certs /etc/istio/egressgateway-ca-certs
```

Comment:
Seems like some change was introduced in Istio against automatic secret watching for `istio-egressgateway-certs` and `istio-egressgateway-ca-certs`. Because I tried this example with latest build and I'm not getting any mounted certs. Maybe we should just keep the deployment patch example entirely.

Reply:
Yes, as I wrote in https://github.com/istio/istio.io/pull/7754#discussion_r457553335